### PR TITLE
Add --purge option to "yunohost app remove"

### PR DESCRIPTION
**Pull request by sagessylu**
- Add --purge option to "yunohost app remove" (https://github.com/YunoHost/issues/issues/1854)
- (no webadmin part)

### CODE REVIEW

**Review by zamentur — APPROVED** [review]
LGTM untested

**Comment by zamentur** [comment]
This PR could be merged, however if we create the webadmin part we need to add a way to know if an app support this feature or not

File: data/actionsmap/yunohost.yml [hunk 1]
```diff
@@ -673,7 +673,11 @@ app:
             api: DELETE /apps/<app>
             arguments:
                 app:
-                    help: App to delete
+                    help: App to remove
+                -p:
+                    full: --purge
+                    help: Remove with all app data
```
Comment (alexAubin) [inline data/actionsmap/yunohost.yml:679]:
```suggestion
                    help: Also remove all application data
```

**Review by alexAubin — APPROVED** [review]
LGTM, cheers ! :+1:

**Pull request merged**